### ISSUE TITLE
Check cabal outdated

### ISSUE THREAD

**Issue by turion** [turion/rhine-koans#16]
There should be a regular CI run that checks `cabal outdated` whether all the dependencies (in particular `rhine` itself) are up to date.

**Comment by endgame** [comment]
Hackage supports dependency notifications, configurable at your user page, which I assume is https://hackage.haskell.org/user/turion/notify .

**Comment by turion** [comment]
That's true, but they don't seem to work for me. I've configured them, but never received a mail. Do they for you?

**Comment by endgame** [comment]
I have email notifications turned off, which might be blocking them for me. I haven't investigated the feature thoroughly, but wanted to flag it in case you weren't already aware.
